src/cmd/policy.go

```diff
@@ -84,6 +98,39 @@ func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return nil, nil
 }
 
+func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term, error) {
+
+	var org, repo string
+
+	if err := ast.As(a.Value, &org); err != nil {
+		return nil, err
+	} else if ast.As(b.Value, &repo); err != nil {
+		return nil, err
+	}
+
+	flags := runCmd.Flags()
+	githubToken, err := flags.GetString("github-token")
+	if githubToken == "" {
+		githubToken = os.Getenv("GITHUB_API_TOKEN")
+	}
+	cobra.CheckErr(err)
+
+	response, err := getClientRest().R().
+		SetHeader("Accept", "application/vnd.github+json").
+		SetHeader("Authorization", fmt.Sprintf("token %s", githubToken)).
```

Comment:
for readability (and since the line is already very long) i would move the 2 `fmt.Sprintf` out to variables and just pump the variables in here.

Reply:
That's a good point.  I'll do that.